elegant/clean_timepoint_data.py

```diff
@@ -187,3 +187,26 @@ def remove_dead_timepoints(experiment_root, postmortem_timepoints, dry_run=False
             timepoints_after_death = timepoint_num - death_timepoint_index # positive values for timepoints after death
             if timepoints_after_death > postmortem_timepoints:
                 remove_timepoint_for_position(experiment_root, position, timepoint, dry_run=dry_run)
+
+def purge_images_from_experiment(experiment_root):
+    '''Removes all image files from experiment directory
+
+    This function is intended to be run after an experiment directory has been backed up
+    (or possibly after an issue with the experiment makes the data non-usable).
+    The function will remove all image files from position directories, while leaving
+    position_metadata files intact.
+
+    Parameters:
+        experiment_root: str/pathlib.Path to experiment root
+    '''
+    delete_confirmed = input(f'Removing all image files from {experiment_root}. Press y and enter to confirm; press enter to abort:')
```

Comment:
Check out how the rest of the functions in `clean_timepoint_data` handle potentially destructive actions: they provide a `dry_run` parameter that you can use to check the actions first. You might want to make `dry_run=True` the default here. 

Even given that it might make sense to request a manual OK, but also having a parameter to override (so the function can be scripted, e.g. `force=False` would be good).

Reply:
Brought into spec with the other file removing functions by including the dry_run kw (default True). Correspondingly elected to remove the prompt.